src/java.base/unix/classes/java/lang/ProcessImpl.java

```diff
@@ -576,7 +574,7 @@ public String toString() {
     // SapMachine 2024-07-01: process group extension
     private static native int terminateProcessGroup(long pid, boolean force);
 
-    private void terminateProcessGroup(boolean force) throws IOException {
+    protected void terminateProcessGroup(boolean force) throws IOException {
```

Comment:
Do you really need to make it protected? I would have thought making it package private would be enough.

Reply:
Good catch. Fixed.